### DISCUSSION TITLE
Fixing broken artwork links

### DISCUSSION THREAD

**Discussion by anony-0** [ChatTheatre/Community#10] [General]
Some questions in regard to how the client is grabbing artwork --

Are we supposed to edit files outside of woe in order to get the client to play nice with the new links?

For instance, my side art panels are missing in orchil -- inspection brings me to this --
 #left {
    background-image: url(http://www.skotos.net/LazarusGame/images/edgeleft.jpg);
}

Which isn't so good, as that is going to the now dead skotos link. So that obviously explains why my side panels are missing.

It's in lazarus.css, but the current  orchil github for that is actually linked instead to multiverse (which makes things nice and confusing) ... so where is orchil picking this data from? Are we supposed to edit data like that here in github or somewhere in woe?

Or am I jumping the gun and should just wait for Shannon?

**Comment by shannona** [comment]
Generally all the image files are on /var/www/html/images on each machine, which is images.domain.

EG: https://images.lazarus-project.net/edgeleft.jpg

**Reply by anony-0** [reply]
> Generally all the image files are on /var/www/html/images on each machine, which is images.domain.
> 
> EG: https://images.lazarus-project.net/edgeleft.jpg

Yep, assumed it was there.  And that address is what I have been using to fix all my old broken map links. But how do I fix orchil so it looks for edgeleft at images.lazarus-project.net? Same with edgeright?

The theatre property doesn't seem to be handling that. And I don't see a woe object that handles it (not saying it's not there somewhere, just that I can't seem to find it). Even the github here seems to be linking instead to multiverse. So... unsure where it is....

I also have the little arrows issue (which CM is having the same issue with) with broken links that I can't find the location for. Arrows are on the image domain, and strangely zealotry can find them, but orchil can't.

**Comment by shannona** [comment]
If there's anything that's missing, let me know ASAP, as I'm closing out Skotos work before the end of the year.

**Comment by ChristopherA** [comment]
@saraht45 Were you able to get the base url for Theatre's to work with Allegory?

**Comment by shannona** [comment]
Orchil files are all in /var/www/html/client. There's a subdirectory for
each game, so for example "lazarus".  The particular things you're looking
for are in the lazarus.css file. So:
/var/www/html/client/lazarus/lazarus.css

I actually fixed that for the other games, but missed yours since we rolled
it out so much later. But, take a look and see if it makes sense to you. If
not, let me know.

I don't know what's  up with the arrows, but if you can find the exact name
(and/or URL that Zealotry is useding) we should be able to track them down.

**Comment by anony-0** [comment]
I'm seeing the css file by simply inspecting it in the browser --  http://client.lazarus-project.net/lazarus/lazarus.css

Although I have no idea how to fix it myself. It still has the skotos links for the artwork.

Am I supposed to be using certain editing tools to find most of these files? Remember I don't have the linode yet, so when files are referenced, I have no idea how to even get to them really. 
I assume once I have the game transferred to my linode acct. I can see and edit the files, add/delete artwork, and so on?

And that may seem like a stupid question, not sure... just remember I am used to simply wrestling with merry in woe, not server , client or underlying code issues.  When told: /var/www/html/client/lazarus/lazarus.css  --it feels like I should have some directory access somewhere that I simply am unaware of.


As for the arrows, zealotry has them at: https://images.lazarus-project.net/arrows/w-ex.gif 
Which is where they should be. I expect orchil is looking for them at a skotos address?
